samples_Run2UL_2017.json

```diff
@@ -651,380 +690,8 @@
-                    "/DoubleEG/Run2017D-UL2017_MiniAODv2-v1/MINIAOD"
-                ]
-            },
-            {
-                "das_names": [
-                    "/DoubleEG/Run2017E-UL2017_MiniAODv2_NanoAODv9-v1/NANOAOD"
-                ],
-                "metadata": {
-                    "year": "2017",
-                    "isMC": false,
-                    "primaryDataset": "DoubleEle",
-                    "era": "E"
-                },
-                "das_parents_names": [
-                    "/DoubleEG/Run2017E-UL2017_MiniAODv2-v1/MINIAOD"
-                ]
-            },
-            {
-                "das_names": [
-                    "/DoubleEG/Run2017F-UL2017_MiniAODv2_NanoAODv9-v1/NANOAOD"
-                ],
-                "metadata": {
-                    "year": "2017",
-                    "isMC": false,
-                    "primaryDataset": "DoubleEle",
-                    "era": "F"
-                },
-                "das_parents_names": [
-                    "/DoubleEG/Run2017F-UL2017_MiniAODv2-v1/MINIAOD"
-                ]
-            }
-	]
-    },
 
-    "DATA_MET": {
-        "sample": "DATA_MET",
-        "json_output": "datasets/Run2UL2017_DATA.json",
 
-        "files": [
-            {
-                "das_names": [
-		    "/MET/Run2017B-UL2017_MiniAODv2_NanoAODv9-v1/NANOAOD"
-                ],
-                "metadata": {
-                    "year": "2017",
-                    "isMC": false,
-                    "primaryDataset": "MET",
-                    "era": "B"
-                }
-            },
-	    {
-                "das_names": [
-                    "/MET/Run2017C-UL2017_MiniAODv2_NanoAODv9-v1/NANOAOD"
-                ],
-                "metadata": {
-                    "year": "2017",
-                    "isMC": false,
-                    "primaryDataset": "MET",
-                    "era": "C"
-                }
-	    },
-	    {
-                "das_names": [
-                    "/MET/Run2017D-UL2017_MiniAODv2_NanoAODv9-v1/NANOAOD"
-                ],
-                "metadata": {
-                    "year": "2017",
-                    "isMC": false,
-                    "primaryDataset": "MET",
-                    "era": "D"
-                }
-	    },
-	    {
-                "das_names": [
-                    "/MET/Run2017E-UL2017_MiniAODv2_NanoAODv9-v1/NANOAOD"
-                ],
-                "metadata": {
-                    "year": "2017",
-                    "isMC": false,
-                    "primaryDataset": "MET",
-                    "era": "E"
-                }
-	    },
-	    {
-                "das_names": [
-                    "/MET/Run2017F-UL2017_MiniAODv2_NanoAODv9-v1/NANOAOD"
-                ],
-                "metadata": {
-                    "year": "2017",
-                    "isMC": false,
-                    "primaryDataset": "MET",
-                    "era": "F"
-                }
-	    }
-	]
-    }
```

Comment:
Why the Data samples are removed?

Reply:
This one is not addressed

Reply:
Does not need to apply changes here. One can surely keep the old structure.

Reply:
This is still not fixed. Could you revert the changes relater to data samples

Reply:
Reverted the changes. Hope it looks better now